actions/push-to-gar-docker/action.yaml

```diff
@@ -183,6 +192,7 @@ runs:
         platforms: ${{ inputs.platforms }}
         ssh: ${{ inputs.ssh }}
         build-contexts: ${{ inputs.build-contexts }}
+        secrets: ${{ inputs.buildkit_secrets }}
```

Comment:
I noticed all other inputs are mapped one-to-one with the docker/build-push-action.

I would prefer if it was the case here as well, since:
```
        context: ${{ inputs.context }}
        build-args: ${{ inputs.build-args }}
        push: ${{ inputs.push == 'true' }}
        tags: ${{ steps.meta.outputs.tags }}
        labels: ${{ inputs.labels }}
        target: ${{ inputs.target }}
        cache-from: ${{ inputs.cache-from }}
        cache-to: ${{ inputs.cache-to }}
        file: ${{ inputs.file }}
        platforms: ${{ inputs.platforms }}
        ssh: ${{ inputs.ssh }}
        build-contexts: ${{ inputs.build-contexts }}
```

The input variable in this case should be `inputs.secrets`